### PR TITLE
Tune metadata in Okta rules to align with the style of other rules

### LINKED ISSUE #490: [Rule Tuning] Tune metadata in Okta rules to align with the style of other rules
## Description

Tune the metadata of our Okta rules so that the `description`, `name`, and `note` fields align with the style of our other rules.

Add additional references to Okta rules to help analysts with triage and investigation.

Tweak the `severity` and `risk_score` values for some Okta rules.

### PR DESCRIPTION
**Pull request by threat-punter**
## Issues
Resolves #490 

## Summary
Tune the metadata of our Okta rules so that the `description`, `name`, and `note` fields align with the style of our other rules.

Add additional references to Okta rules to help analysts with triage and investigation.

Tweak the `severity` and `risk_score` values for some Okta rules.

Separate deactivation, modification, and deletion actions for Okta policy, policy rule, and network zone objects into separate rules.

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/okta/attempt_to_deactivate_okta_network_zone.toml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+[rule]
+author = ["Elastic"]
+description = """
+Detects attempts to deactivate an Okta network zone. Okta network zones can be configured to limit or restrict access to
+a network based on IP addresses or geolocations. An adversary may attempt to modify, delete, or deactivate an Okta
+network zone in order to remove or weaken an organization's security controls.
+"""
+false_positives = [
+    """
+    Consider adding exceptions to this rule to filter false positives if Oyour organization's Okta network zones are
```
Comment (threat-punter) [inline rules/okta/attempt_to_deactivate_okta_network_zone.toml:16]:
```suggestion
    Consider adding exceptions to this rule to filter false positives if your organization's Okta network zones are
```

File: rules/okta/credential_access_attempted_bypass_of_okta_mfa.toml [hunk 2]
```diff
@@ -7,14 +7,14 @@ updated_date = "2020/10/26"
 [rule]
 author = ["Elastic"]
 description = """
-An adversary may attempt to bypass the Okta multi-factor authentication (MFA) policies configured for an organization in
-order to obtain unauthorized access to an application. This rule detects when an Okta MFA bypass attempt occurs.
+Identifies attempted to bypass Okta multi-factor authentication (MFA). An adversary may attempt to bypass the Okta MFA
```
Comment (threat-punter) [inline rules/okta/credential_access_attempted_bypass_of_okta_mfa.toml:10]:
```suggestion
Identifies attempts to bypass Okta multi-factor authentication (MFA). An adversary may attempt to bypass the Okta MFA
```

**Review by Samirbous — APPROVED** [review]

File: rules/okta/credential_access_okta_brute_force_or_password_spraying.toml [hunk 3]
```diff
@@ -21,7 +21,7 @@ index = ["filebeat-*", "logs-okta*"]
 language = "kuery"
 license = "Elastic License"
 name = "Okta Brute Force or Password Spraying Attack"
-note = "The Okta Filebeat module must be enabled to use this rule."
+note = "The Okta Fleet integration or Filebeat module must be enabled to use this rule."
```
Comment (bm11100) [inline rules/okta/credential_access_okta_brute_force_or_password_spraying.toml:24]: nice

**Review by bm11100 — APPROVED** [review]
Looks good! My only suggestion is for the description sometimes you use `Detects` and sometimes you use `Identifies`. I'd choose just one to use for consistency. 🌟 ‼️

File: rules/okta/credential_access_attempted_bypass_of_okta_mfa.toml [hunk 4]
```diff
@@ -7,14 +7,14 @@ updated_date = "2020/10/26"
 [rule]
 author = ["Elastic"]
 description = """
-An adversary may attempt to bypass the Okta multi-factor authentication (MFA) policies configured for an organization in
-order to obtain unauthorized access to an application. This rule detects when an Okta MFA bypass attempt occurs.
+Identifies attempts to bypass Okta multi-factor authentication (MFA). An adversary may attempt to bypass the Okta MFA
```
Comment (threat-punter) [inline rules/okta/credential_access_attempted_bypass_of_okta_mfa.toml:10]:
```suggestion
Detects attempts to bypass Okta multi-factor authentication (MFA). An adversary may attempt to bypass the Okta MFA
```

File: rules/okta/impact_possible_okta_dos_attack.toml [hunk 5]
```diff
@@ -7,14 +7,14 @@ updated_date = "2020/10/26"
 [rule]
 author = ["Elastic"]
 description = """
-An adversary may attempt to disrupt an organization's business operations by performing a denial of service (DoS) attack
-against its Okta infrastructure.
+Identifies possible Denial of Service (DoS) attacks against an Okta organization. An adversary may attempt to disrupt an
```
Comment (threat-punter) [inline rules/okta/impact_possible_okta_dos_attack.toml:10]:
```suggestion
Detects possible Denial of Service (DoS) attacks against an Okta organization. An adversary may attempt to disrupt an
```

File: rules/okta/okta_attempt_to_modify_okta_policy.toml [hunk 6]
```diff
@@ -7,9 +7,9 @@ updated_date = "2020/10/26"
 [rule]
 author = ["Elastic"]
 description = """
-An adversary may attempt to modify an Okta policy in order to weaken an organization's security controls. For example,
-an adversary may attempt to modify an Okta multi-factor authentication (MFA) policy in order to weaken the
-authentication requirements for user accounts.
+Identifies attempts to modify an Okta policy. An adversary may attempt to modify an Okta policy in order to weaken an
```
Comment (threat-punter) [inline rules/okta/okta_attempt_to_modify_okta_policy.toml:10]:
```suggestion
Detects attempts to modify an Okta policy. An adversary may attempt to modify an Okta policy in order to weaken an
```

File: rules/okta/persistence_attempt_to_reset_mfa_factors_for_okta_user_account.toml [hunk 7]
```diff
@@ -7,8 +7,9 @@ updated_date = "2020/10/26"
 [rule]
 author = ["Elastic"]
 description = """
-An adversary may attempt to remove the multi-factor authentication (MFA) factors registered on an Okta user's account in
-order to register new MFA factors and abuse the account to blend in with normal activity in the victim's environment.
+Identifies attempts to reset an Okta user's enrolled multi-factor authentication (MFA) factors. An adversary may attempt
```
Comment (threat-punter) [inline rules/okta/persistence_attempt_to_reset_mfa_factors_for_okta_user_account.toml:10]:
```suggestion
Detects attempts to reset an Okta user's enrolled multi-factor authentication (MFA) factors. An adversary may attempt
```

File: rules/okta/credential_access_okta_brute_force_or_password_spraying.toml (hunk 3 above)
Comment (threat-punter) [reply rules/okta/credential_access_okta_brute_force_or_password_spraying.toml:24]: 😎

**Comment by threat-punter** [comment]
> Looks good! My only suggestion is for the description sometimes you use `Detects` and sometimes you use `Identifies`. I'd choose just one to use for consistency. 🌟 ‼️

Good catch. Updated to use "Detects"

**Pull request merged**